### ISSUE TITLE
Add oembed-play url images as a setting

### ISSUE THREAD

**Issue by Designscheu** [distantnative/oembed#33]
The current 

    assets/oembed/oembed-play.png

should be a setting to either change the path or image name, e.g.:

    c::set('oembed.playUrl', '/assets/oembed-play.png'

**Comment by distantnative** [comment]
Will be available with the upcoming 2.0.0 (already added to the develop branch but still work-in-progress).
```php
c::set('plugin.oembed.video.lazyload.btn', 'assets/plugins/oembed/images/play.png')
```

**Comment by distantnative** [comment]
2.0.0 just got released. It's a complete rewrite, so it potentially breaks current setups. But an update is worth it - and it allows you easily to replace the play image now ;)
